picasso/src/test/java/com/squareup/picasso/DeferredRequestCreatorTest.java

```diff
@@ -54,48 +50,13 @@
     initMocks(this);
   }
 
-  @Config(reportSdk = 12)
-  @Test public void initAddsAttachListenerWhichDefersLayoutListener() {
```

Comment:
"That said, min SDK of the project went to 14 in the gradle change. Should that still be something older? (I don't know what it was before)"

This is the reason I didnt comment about this. I thought the min version we support was bumped.